ckanext/canada/templates/internal/user/list.html

```diff
@@ -1,43 +1,95 @@
-{% extends 'admin/base.html' %}
+{% if c.userobj and c.userobj.sysadmin %}
```

Comment:
we still need this large template in our extension? Was hoping we could remove some code as part of this change so that we have an easier time upgrading. I'm assuming this is mostly copied from core, right?

Reply:
@wardi Core does not display the user full name or user role. In our template, I am keeping the "User" tab in the ckan-admin structure. Without it, the user list does not display anywhere in the ckan-admin in core as far as I can tell.

Reply:
Do you think this could be submitted as a core feature so we don't have to maintain it in our extension and fix it as core changes?